bridge/src/main/java/com/bridge/renderHandlerImplementation/sprite/Sprite.java

```diff
@@ -0,0 +1,5 @@
+package com.bridge.renderHandlerImplementation.sprite;
+
+public class Sprite {
```

Comment:
All Sprite, Sound, Coord, IFile, etc already exists on the project, in the package piece, this have to be moved inside package render handler

Reply:
done